aws_jobs/cyhy-data-extract.py

```diff
@@ -111,18 +113,23 @@ def cleanup_old_files(output_dir, file_retention_num_days):
 
 
 # TODO Finish function to delete files until there is only X in the bucket
-def cleanup_bucket_files(aws_access_key_id, aws_secret_access_key):
+def cleanup_bucket_files():
```

Comment:
Why do we have both `MAX_ENTRIES` hardcoded above and `FILE_RETENTION_NUM_DAYS` in the config file (used below in `query_data())`?  Is there a reason why we shouldn't just use `FILE_RETENTION_NUM_DAYS` in both places?

Reply:
That comes down to deletion preference. The way it is coded the latest x output archives are saved by filename. So if you create a new archive using a date six months ago it would still be deleted if there were more than x archives in the bucket. If preferred it can be set to sort by last modification date instead and then `FILE_RETENTION_NUM_DAYS` could be used to delete anything older than that instead of capping at a number of objects.